Shogendar.Karikari.Backend/Controllers/EventController.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Shogendar.Karikari.Models;
+
+namespace Shogendar.Karikari.Backend.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EventController : ControllerBase
+    {
+        [HttpGet]
+        public IActionResult ShowEvents(int id){
+            Db db = new();
+            var query = from Event in db.Events
+                        where Event.Id == id
+                        select Event;
+            return Ok(query.ToList());

```

Comment:
[must] 指定したIdを持つ項目がなかった場合は、404応答を返すようにしてください。

`query.Any()`は、ひとつ以上の項目が見つかった場合に真となるので、これを活用するとよいと思います。

```suggestion
            if(!query.Any())
                return NotFound();
            return Ok(query.ToList());

```